SampleViewer/mainSample.cpp

```diff
@@ -302,10 +302,15 @@ int main(int argc, char *argv[])
 #endif // QT_WEBVIEW_WEBENGINE_BACKEND
 
   QGuiApplication::setApplicationName("ArcGIS Maps Qt Samples");
-
   QGuiApplication::setOrganizationName("Esri");
   QApplication app(argc, argv);
-
+  
+  #ifdef ESRI_AUTOMATION_TEST_SUSE_FONT_PATH
+  // There is a known issue on certain Linux distros with newer versions of Qt 
+  // where no fonts are loaded by default. Work around it by loading a specific font
+  QFontDatabase::addApplicationFont(QUOTE(ESRI_AUTOMATION_TEST_SUSE_FONT_PATH));
+  #endif // ESRI_AUTOMATION_TEST_SUSE_FONT
```

Comment:
Should this be an environment variable we read in? If it's set at compile time, @har13205 won't be able to customize the value that is set.

Reply:
Oh right, I missed that. This uses an env variable to get the path, but this is checked in compile time. I'll fix that.